packages/react/src/components/tabs/tabs.tsx

```diff
@@ -126,13 +123,21 @@ interface TabItem extends Tab {
     onBeforeUnload?(): Promise<boolean>;
 }
 
+interface AddButtonProps {
+    label?: string;
+    disabled?: boolean;
+    loading?: boolean;
+    tooltipContent?: string;
+    onClick?(): void;
```

Comment:
Scuse on s'est peut-être mal compris. Faut au minimum le `onClick` pour avoir un bouton valide
```suggestion
    onClick(): void;
```